examples/advanced/edge/simulators/processors/pt_xor_task_processor.py

```diff
@@ -48,7 +48,7 @@ def setup(self, device_info: DeviceInfo, user_info: UserInfo, job: JobResponse)
         self.user_info = user_info
         self.job_id = job.job_id
         self.job_name = job.job_name
-        device_io_dir = f"./workspace_emulator/{device_info.device_id}_output"
+        device_io_dir = f"/tmp/nvflare/workspaces/edge_simulator_xor/{device_info.device_id}"
```

Comment:
where does this "edge_simulator_xor" come from?

Reply:
this is for saving the local training curves

Reply:
for saving local training curves